google_drive.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_dependency('nokogiri', ['>= 1.5.3', '< 2.0.0'])
-  s.add_dependency('google-api-client', ['>= 0.11.0', '< 0.22.0'])
+  s.add_dependency('google-api-client', ['>= 0.11.0', '< 0.25.0'])
```

Comment:
You specify '< 0.25.0' here which means that 0.25.x is still not accepted. Is that OK?

Reply:
Yes, it works for me, 

on a side not 0.25.0 is the latest version  https://rubygems.org/gems/google-api-client/versions/0.25.0

Reply:
I will change it to `=<` to be sure

Reply:
changed it to `'google-api-client', ['>= 0.11.0', '< 0.25.1']` so people can get the latest version

Reply:
it resolved to `google-api-client 0.25.0`